app/scripts/auth.js

```diff
@@ -8,7 +8,12 @@ const redirectPath = '/login'
 
 /* eslint-disable new-cap */
 export const isAuthenticated = connectedReduxRedirect({
-  authenticatedSelector: state => state.auth.isAuthenticated,
+  // return true statements if festival is free
+  authenticatedSelector: state => {
+    return (
+      (state.auth.isAuthenticated || state.meta.config.festivalTicketPrice === 0)
```

Comment:
When using the `isAuthenticated` method you might really just think it allows only users which are authenticated. Adding another permission which is dependend on a config var / free festival can be confusing and lead to bugs  (eg. why would I think that `isAuthenticated` is dependend on the festival price?).

As I can see now the `/profiles` route would be readable just because the festival price is 0? Here: https://github.com/adzialocha/hoffnung3000/blob/43b3bccbe18ff01563eeef7f009e74eb70bef145/app/scripts/routes.js#L56

The `isParticipantOrFreeFestival` is better as its more clear what it implies. I suggest doing something similar here!

Reply:
fixed here https://github.com/adzialocha/hoffnung3000/commit/c98d4e5caa3a794b630d5447aca5bd514c00980f